pkg/api/types.go

```diff
@@ -1146,10 +1146,97 @@ const (
 )
 
 // Affinity is a group of affinity scheduling rules, currently
-// only node affinity, but in the future also inter-pod affinity.
+// includes node affinity,pod affinity & pod anti affinity.
```

Comment:
I don't think we're planning to add any more types of affinity here, so I think you can remove everything in the comment after the word "rules"